pkg/apis/litmuschaos/v1alpha1/chaosexperiment_types.go

```diff
@@ -73,7 +73,7 @@ type ExperimentDef struct {
 	// List of Permission needed for a service account to execute experiment
 	Permissions []rbacV1.PolicyRule `json:"permissions"`
 	// List of ENV vars passed to executor pod
```

Comment:
Makes sense to call this chaosexperiment pod

Reply:
you mean it's this? https://github.com/litmuschaos/chaos-operator/pull/309/commits/659798a2ebfde7e2fa09369188640790f2ec5804